docs/spec/govuk_tech_docs/open_api/renderer_spec.rb

```diff
@@ -1,244 +0,0 @@
-# frozen_string_literal: true
```

Comment:
not sure we should be deleting this spec?

Reply:
Whoops, no, I'll re-instate it